docs/standard/serialization/system-text-json-handle-overflow.md

```diff
@@ -82,6 +82,16 @@ When the target object is serialized, the extension data key value pairs become
 
 Notice that the `ExtensionData` property name doesn't appear in the JSON. This behavior lets the JSON make a round trip without losing any extra data that otherwise wouldn't be deserialized.
 
+The following example shows a round trip from JSON to a deserialized object and back to JSON:
+
+:::code language="csharp" source="snippets/system-text-json-how-to/csharp/RoundtripExtensionData.cs" highlight="13-14":::
+
+## Deserialize into JsonElement
+
+If you just want to be flexible about what JSON to accept for a particular property, an alternative is to deserialize into <xref:System.Text.Json.JsonElement>. Any valid JSON property can be deserialized into `JsonElement`. The following example shows a round trip from JSON and back to JSON for a class that includes properties of type `JsonElement`.
```

Comment:
Should we create a .NET 6.0 partition and mention the `JsonNode` type? `JsonElement` would be used if immutability is okay, whereas `JsonNode` offers mutability.

Reply:
I'll create a new issue for this and include it in the PR that creates the .NET 6 Preview zone pivot.